src/styles/blocks/button.scss

```diff
@@ -0,0 +1,28 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 100%;
+  height: 56px;
+  box-sizing: border-box;
+
+  font-size: 16px;
+  font-family: inherit;
+  text-decoration: none;
+  font-weight: $text-bold;
+  background-color: $color-main-red;
+  color: $color-white;
+  border: none;
+
+  cursor: pointer;
+  transition: all 0.3s;
```

Comment:
0.3s this is something that is used all over the styles. It would be better to move such values to variables as well